src/Dummy.cpp

```diff
@@ -10,6 +10,8 @@ const Dummy::uid_t Dummy::Join(const std::string &addr) {
   ++user_count;
   uid2addr.emplace(LastPlayer(), addr);
   holecards.emplace(LastPlayer(), std::vector<card_t>());
+  bankroll.emplace(LastPlayer(), 0);
```

Comment:
curious,  couldn't you make the player as a complicate class, 
e.g. 
class player {
    mUid
    mbankRoll
    roundBets
}